lib/main_common.pm

```diff
@@ -45,6 +46,10 @@ our @EXPORT = qw(
   init_main
   installyaststep_is_applicable
   installzdupstep_is_applicable
+  is_s390x
```

Comment:
Utils::Architectures already is defining these symbols. main_common doesn't need to export them.

Reply:
yes, thanks